repository/src/main/java/com/epam/esm/exceptions/DataNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.esm.exceptions;
+
+import lombok.Getter;
+
+@Getter
```

Comment:
It would be better if you add javadoc for exception to describe when and where they are used